display/d.labels/do_labels.c

```diff
@@ -463,6 +464,8 @@ int scan_ref(char *buf)
         FALLTHROUGH;
     default:
         return 0;
+    case EOF:
+        return 0;
```

Comment:
`case EOF` should be the placed as the first case after the switch statement. Alternatively, before `default`, but with `FALLTHROUGH;` instead of `return 0;`

The `default` would be the outcome even without a specific `case EOF`, so this change only clarifies the intention.

Reply:
did changes but CI check is failing